test/integration/serverTest.js

```diff
@@ -225,6 +225,26 @@ describe('Brave Sensor server', () => {
       })
     })
 
+    it('should send a 200 response if a sirenAddressed call is receieved', async () => {
+      await firmwareAlert(radar_coreID, SENSOR_EVENT.STILLNESS)
+      const response = await chai.request(server).post('/api/sirenAddressed').send({})
+      expect(response).to.have.status(200)
+    })
+
+    it('should send a 200 response if a sirenEscalated call is receieved', async () => {
+      await firmwareAlert(radar_coreID, SENSOR_EVENT.STILLNESS)
+      const response = await chai.request(server).post('/api/sirenEscalated').send({})
+      expect(response).to.have.status(200)
+    })
+
+    it('should start an alert session if a sirenEscalated call is receieved', async () => {
+      await firmwareAlert(radar_coreID, SENSOR_EVENT.STILLNESS)
+      await chai.request(server).post('/api/sirenEscalated').send({})
+      const sessions = await db.getAllSessionsFromLocation(testLocation1Id)
+      const session = sessions[sessions.length - 1]
+      expect(session.alertType).to.equal(ALERT_TYPE.SENSOR_STILLNESS)
+    })
```

Comment:
All of these tests are going to get a validation error because you aren't sending `coreid` in the body.  I suggest making new helper functions for calling `POST /api/sirenAddressed` and `POST /api/sirenEscalated` similar to the `firmwareAlert` function.

It will also be good to have a test to see that the state was actually changed to `COMPLETED` when you call `POST /api/sirenAddressed`.

Reply:
I'm confused what function body I need to send coreid in. Radar_coreID is defined in the beforeEach section.

Reply:
It needs to be set in the body that is being sent to `POST /api/sirenEscalated`. If you look in `firmwareAlert` you can see how Mario's sending a JSON object with `coreid` and other things to `POST /api/sensorEvent`.

Reply:
What function body does he coreid need be included in?

Reply:
Inside the curly braces of the `send({})` in  `await chai.request(server).post('/api/sirenEscalated').send({})`